### DISCUSSION TITLE
how to use the remaining disks on the nodes to expand the cluster,

### DISCUSSION THREAD

**Discussion by ming12713** [minio/minio#17360] [Q&A] [answered]
Hi
we initializing the cluster, only 6 disks on the nodes are used with the configuration `MINIO_VOLUMES="http://los{1...3}.test.com:9000/mnt/disk{1...6}/minio".`
Now, we want to scale up the cluster and add the remaining disks from the nodes, how can i do it ?

**Comment by dormanze** [comment, marked as answer]
You can refer to the following doc:
https://min.io/docs/minio/kubernetes/upstream/operations/concepts.html#can-i-change-the-size-of-an-existing-minio-deployment
